src/Connection/ConnectionInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Connection;
+
+use Yiisoft\Db\Command\Command;
+use Yiisoft\Db\Exception\Exception;
+use Yiisoft\Db\Exception\InvalidConfigException;
+use Yiisoft\Db\Schema\Schema;
+
+interface ConnectionInterface
```

Comment:
Should it contain more methods such as the following?

- beginTransaction(), getTransaction()
- open(), close(), isActive()